Bottom: cefe48442ce9b44d56b41acf3dfb6b683bed97a2
Top:    4fb8993653d9a6ab68fcba49c30feb676dc4e815
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-14 01:37:46 +0100

Refresh of hack-to-make-pretty-translucen

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d3e5e97..4999005 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -576,9 +576,9 @@ void
 ghid_thindraw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 {
   common_thindraw_pcb_polygon (gc, poly, clip_box);
-  ghid_global_alpha_mult (gc, 0.25);
+  ghid_set_alpha_mult (gc, 0.25);
   ghid_fill_pcb_polygon (gc, poly, clip_box);
-  ghid_global_alpha_mult (gc, 1.0);
+  ghid_set_alpha_mult (gc, 1.0);
 }
 
 void
